java-client/session/src/main/java/io/deephaven/client/impl/UpdateByBuilder.java

```diff
@@ -118,6 +118,18 @@ public UpdateByColumn.UpdateBySpec visit(EmaSpec ema) {
                     .build();
         }
 
+        // TODO: complete properly (DHC ticket #3666)
```

Comment:
I think this is the wrong issue number? Should these be #3392?

Reply:
I am trying to close #3392 since it covers operators already merged.